Makefile

```diff
@@ -12,6 +12,8 @@ export TERRAFORM_PROVIDER_REPO ?= https://github.com/hashicorp/terraform-provide
 export TERRAFORM_DOCS_PATH ?= website/docs/r
 export PROVIDER_NAME
 
+export UXP_VERSION := 1.14.6-up.1
```

Comment:
Since the UXP version is currently [pinned](https://github.com/upbound/provider-aws/blob/9bc813763cb4ec29ca464cb375af530f259c5abc/Makefile#L71) in the main branch, you can continue by rebase after removing this line.

Reply:
Done.